server/server.js

```diff
@@ -10,26 +10,6 @@ function server(app) {
       handlers: notificationsConfiguration
     })
   );
-  app.use(
-    "/batch",
-    notifHandler({
```

Comment:
Now we will be going through smart-notifier, the code is different. Is `smartNotifierHandler` able to handle what `notifHandler` does for batches?